packages/babel-types/src/modifications/typescript/removeTypeDuplicates.ts

```diff
@@ -53,13 +53,16 @@ export default function removeTypeDuplicates(
   }
 
   // add back in bases
-  for (const type of Object.keys(bases)) {
+  for (const type of Object.keys(bases) as (keyof typeof bases)[]) {
     types.push(bases[type]);
   }
 
   // add back in generics
   for (const name of Object.keys(generics)) {
-    types.push(generics[name]);
+    types.push(
+      // @ts-ignore generics are not implemented
```

Comment:
The generics logic is implemented in https://github.com/babel/babel/pull/14393, I will revive that PR after current `noImplicitAny` works.